docs/platform/webcontainers/browser-support.md

```diff
@@ -64,6 +64,8 @@ We have beta support for Firefox. Please try it and [share feedback with us](htt
 
 Firefox does not fully support the required mode for cross-origin isolation, so you might encounter limitations when **running a server in a preview frame** within the StackBlitz editor. Third-party assets might get blocked due to the limitations of cross-origin isolation policies. However, you can work around this by opening your server preview in a separate window.
 
+If you are using Firefox Private Browsing, it's not possible to run a webserver. The reason is that we need to be able to install a ServiceWorker which is not possible in a private window.
```

Comment:
Practically, what does this look like? When I ran my tests it seemed like "the preview window won't load" would be the most common way to encounter this problem? Is that true?

Reply:
Currently on production, you will see this message in the preview window

![image](https://github.com/stackblitz/docs/assets/1913805/d5e4a127-9f8f-43c8-b3bf-0ce5387d9e55)

I created a PR to add a custom error screen for this specific scenario https://github.com/stackblitz/stackblitz/pull/3562.

The reason is that WebContainer internally can't install the SW which is necessary to run web servers. You can still run `npm install` and what not.

In Codeflow however, extensions that use web views are broken as well for instance.